### ISSUE TITLE
summary method: odds ratio as summary option

### ISSUE THREAD

**Issue by tercer** [IQSS/Zelig#141]
Add an option to specify odds ratios instead of estimated parameter in the summary method for the model estimate, for certain choice models (particularly the Logit family) where that is applicable.

**Comment by adamobeng** [comment]
I can take a crack at this. 

Would the right way to do this would be to modify the `show` method in `model-zelig.R`, or to modify `z$out` in each of the model types' classes?

**Comment by cchoirat** [comment]
Great!  You may want to try to overwrite the show method in model-logit.R.

**Comment by adamobeng** [comment]
I've added an argument called show.odds.ratios, implemented it in the way you suggest (branch [feature/odds_ratio](https://github.com/adamobeng/Zelig/tree/feature/odds_ratio))

It works like this:

```{r}
> mydata <- read.csv("http://www.ats.ucla.edu/stat/data/binary.csv")
> z5 <- zlogit$new()
> z5$zelig(admit ~ gre + gpa, data = mydata, epsilon=1)
> z5$summarize()
Model: 

Call:
z5$zelig(formula = admit ~ gre + gpa, data = mydata, epsilon = 1)

Deviance Residuals: 
    Min       1Q   Median       3Q      Max  
-1.2360  -0.8699  -0.6970   1.3372   2.0916  

Coefficients:
             Estimate Std. Error z value Pr(>|z|)
(Intercept) -4.999760   1.057053  -4.730 2.25e-06
gre          0.002670   0.001084   2.463   0.0138
gpa          0.750189   0.329060   2.280   0.0226

(Dispersion parameter for binomial family taken to be 1)

    Null deviance: 499.98  on 399  degrees of freedom
Residual deviance: 480.85  on 397  degrees of freedom
AIC: 486.85

Number of Fisher Scoring iterations: 1

Next step: Use 'setx' method
> z5$summarize(show.odds.ratios=T)
Model: 

Call:
z5$zelig(formula = admit ~ gre + gpa, data = mydata, epsilon = 1)

Deviance Residuals: 
    Min       1Q   Median       3Q      Max  
-1.2360  -0.8699  -0.6970   1.3372   2.0916  

Coefficients:
            Estimate (odds ratio) Std. Error (odds ratio) z value Pr(>|z|)    
(Intercept)               0.00674                 2.87788  -4.730 2.25e-06 ***
gre                       1.00267                 1.00109   2.463   0.0138 *  
gpa                       2.11740                 1.38966   2.280   0.0226 *  
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1

(Dispersion parameter for binomial family taken to be 1)

    Null deviance: 499.98  on 399  degrees of freedom
Residual deviance: 480.85  on 397  degrees of freedom
AIC: 486.85

Number of Fisher Scoring iterations: 1
```

Is that what you expected?

Also, am I right in the assumptions that:
- displaying odds ratios for coefficients is only relevant if `.self$mi` and `.self$boostrap` are false?
- I can ignore the cases when .self$by is not "by"?

**Comment by christophergandrud** [comment]
Thanks @adamobeng for putting this together!

I'm wondering if this belongs in the main `show` method. That way it can be more easily applied to other relevant model types. Does that make sense @tercer?

**Comment by christophergandrud** [comment]
Hi @adamobeng Would you be willing to make this a pull request. I'ld like to consider it for an upcoming Zelig release

**Comment by christophergandrud** [comment]
Merged in 3c13ff0a02eb334539c603df453993ee2f802345

A few quick notes:

I shortened the argument to `odds_ratios`. In the output `(OR)` rather than `(odds ratio)` also for brevity.

The commit also documents the argument in the vignette and adds a test.
